exercises/ocr-numbers/.meta/hints.md

```diff
@@ -0,0 +1,3 @@
+## Hints
+
+ocr.convert should validate its input and raise ValueErrors with meaningful error messages if necessary.
```

Comment:
We should probably chance that to something like:

> "Your converter should validate its input and raise a ValueError with a meaningful message if it receives a string that isn't a valid OCR number."

If we mention the "convert" name we're sort of assuming the property name will never change in the canonical data.

Of course if you change this you'll need to rebuild the README.md again as well.

Reply:
I changed that.